hubmap/frontend/cypress/integration/stub_globus_auth.js

```diff
@@ -0,0 +1,40 @@
+describe('The Login Page', function () {
+    before(() => {
+      Cypress.Cookies.debug(true)
+      cy.server();
+      const api = 'http://localhost:8000/';
+      cy.route(`${api}/api/?format=json`, 'fixture:base.json');
+      cy.route(`${api}/api/colors/?format=json`, 'fixture:colors.json');
+      cy.route(`${api}/api/genes/?format=json`, 'fixture:genes.json');
```

Comment:
This is getting copied and pasted a lot: `before` is meant to set up a common starting environment for tests, and we should use it that way.

Reply:
yes that is the idea behind this since we have bunch of redirects and stubs even. The reason I spent a whole day on this is to **learn** cypress and work, am aware when before is used, that is the intention behind having before. @mccalluc what do you specifically mean by **copied** how do you know if it is **needed** or **copied** blindly as you are indicating ?

Reply:
yes that is the idea behind this since we have bunch of redirects and stubs even. The reason I spent a whole day on this is to learn cypress and work, am aware when before is used, that is the intention behind having before. @mccalluc it worries me when you imply *copied everywhere* as you seem to be implying blind copy without knowing what before is ? Are you asking how I came up with before or if I understand concept of before at all ?